src/xclim/core/indicator.py

```diff
@@ -1311,7 +1311,7 @@ def _format(
         # Add formatting {} around values to be able to replace them with _attrs_mapping using format.
         for k, v in args.items():
             if isinstance(v, units.Quantity):
-                mba[k] = f"{v:g~P}"
+                mba[k] = f"{v:gcf}"
```

Comment:
I'm sure this works, but I'm not clear what the change here is.

Reply:
The units are formatted with the `cf` formatter (defined in `cf-xarray`), instead of the short (`~`) pretty (`P`) formatter.  They look a lot alike, the main difference being in how they express negative powers / fractions.